domain/user/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bootstrap
+
+import (
+	"context"
+
+	"github.com/canonical/sqlair"
+	"github.com/juju/errors"
+
+	"github.com/juju/juju/core/database"
+	"github.com/juju/juju/core/user"
+	"github.com/juju/juju/domain/user/state"
+)
+
+const adminUserName = "admin"
+const adminUserDisplayName = "admin"
+
+// GenerateAdminUser inserts the admin user into database.
+// It is used to bootstrap the database.
+//
+// Admin user is created with the following characteristics:
+// 1. This is first user created in the system.
+// 2. This user is used to owner of the first model created in the system.
+// 3. This user is created with no password authorization by default.
+func GenerateAdminUser() func(context.Context, database.TxnRunner) error {
+	return func(ctx context.Context, db database.TxnRunner) error {
+		adminUserUUID, err := user.NewUUID()
```

Comment:
So based on the previous PR feed back this is what I think needs to happen here.

`adminUserUUID` needs to be created in the top level function of `GenerateAdminUser` and returned here instead of inside the txn runner func. This is so bootstrap can get access to the admin uuid and use it as input into other processors later.